Corona-Warn-App/src/main/res/values-de/release_info_strings.xml

```diff
@@ -7,7 +7,7 @@
     <!-- XHED: Title for the release info screen -->
     <string name="release_info_header">"Neue Funktionen"</string>
     <!-- XHED: Version title for the release info screen -->
-    <string name="release_info_version_title">"Release <xliff:g id="release_app_version" example="1.12">"%1$d"</xliff:g></string>
+    <string name="release_info_version_title">"Release <xliff:g id="release_app_version" example="1.12">"%1$s"</xliff:g></string>
```

Comment:
I think this isn't needed or at least we don't use it everywhere else: `xliff:g`

Reply:
I thought so too, but I included it because I've found it in multiple places through our strings and wasn't sure if this was the convention or not. I will remove it.